operations/commit_queue.go

```diff
@@ -202,6 +202,10 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 	grip.Infof("Queue Length: %d\n", len(cq.Queue))
 	for i, item := range cq.Queue {
 		grip.Infof("%d:", i+1)
+		author, _ := client.GetCommitQueueItemAuthor(ctx, projectID, restModel.FromAPIString(item.Issue))
```

Comment:
Not logging this error could make it difficult to debug.

Reply:
If I log this with grip.Error() will command line users see the error? I'd rather they not